src/templates/template_stage_copy_table_multithread.py

```
@@ -0,0 +1,81 @@
+import time
+from src.utils.log.log_utils import LogUtils
+from src.utils.table.table_manager import TableManager
+from sqlalchemy.engine import Engine as _Engine
+from src.utils.table.table_manager import TableManager
```

Ruff (0.12.2)

Redefinition of unused `TableManager` from line 3

Remove definition: `TableManager`

(F811)